.github/workflows/rust.yml

```diff
@@ -103,3 +103,44 @@ jobs:
 
     - name: Package (publish)
       run: cargo +nightly publish --dry-run --target=wasm32-unknown-unknown
+
+  bazel:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Mount Bazel cache
+      uses: actions/cache@v2
+      with:
+        path: "${HOME}/.cache/bazel"
+        key: bazel
+
+    - name: Install Bazelisk
+      run: |
+        curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.5.0/bazelisk-linux-amd64"
+        echo "605c4b417654a09febc7e15c6b802195fdd1c4fe8d876145fb15f10d4fa27b5b  bazelisk-linux-amd64" | sha256sum --check
+        mkdir -p "${HOME}/bin/"
+        mv bazelisk-linux-amd64 "${HOME}/bin/bazel"
+        chmod +x "${HOME}/bin/bazel"
```

Comment:
Bazelisk 1.5.0 is already installed in GHA ubuntu-latest, You just need an alias. https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
I should be able to just call it directly, shouldn't it?

Done.